### PR TITLE
Art.validate page size

### PR DESCRIPTION
**Pull request by ATeal**
## Description of change
There have been issues with large page sizes and downstream systems. This is the beginning of solving that issue by validating page size from pdfinfo

resolves https://github.com/department-of-veterans-affairs/vets-contrib/issues/2846

## Testing done
- rspec

#### Applies to all PRs

- [x] Appropriate logging
- [x] Swagger docs have been updated, if applicable
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)
- [x] Provide which alerts would indicate a problem with this functionality (if applicable)

### CODE REVIEW

File: modules/vba_documents/app/swagger/vba_documents/document_upload/status_code_description.md [hunk 1]
```diff
@@ -7,5 +7,6 @@ Unambiguous status code. Only present if status = "error"
 * `DOC105` - Invalid or unknown id
 * `DOC106` - File size limit exceeded. Each document may be a maximum of 100MB.
 * `DOC107` - Empty payload.
+* `DOC108` - Maxium dimensions exceeded. Height and width must be less than 21 in x 21 in.
```
Comment (bastosmichael) [inline modules/vba_documents/app/swagger/vba_documents/document_upload/status_code_description.md]: Was this ever decided on as a breaking change or not? Before we merge we need to confirm with Andrew on whether or not VetPro and Vetrespec are okay with this.

File: lib/pdf_info.rb [hunk 2]
```diff
@@ -42,8 +42,30 @@ def pages
...
+      in_pts = page_size
+      {
+        height: pts_to_inches(in_pts[:height]),
+        width: pts_to_inches(in_pts[:width])
+      }
+    end
+
     private
 
+    def pts_to_inches(dimension)
```
Comment (bastosmichael) [inline lib/pdf_info.rb]: Can you describe this method a little better?

File: modules/vba_documents/lib/vba_documents/upload_error.rb [hunk 3]
```diff
@@ -13,6 +13,7 @@ class UploadError < StandardError
     DOC105 = 'Invalid or unknown id'
     DOC106 = 'Maximum document size exceeded. Limit is 100MB per document'
     DOC107 = 'Empty payload'
+    DOC108 = 'Maximum page size exceeded'
```
Comment (bastosmichael) [inline modules/vba_documents/lib/vba_documents/upload_error.rb]: We should probably do like DOC106 and tell them what the size is in the error response.

File: modules/vba_documents/app/swagger/vba_documents/document_upload/status_code_description.md (hunk 1 above)
Comment (ATeal) [reply modules/vba_documents/app/swagger/vba_documents/document_upload/status_code_description.md]: Yes in the ticket they mentioned the communication

Comment (ATeal) [reply modules/vba_documents/app/swagger/vba_documents/document_upload/status_code_description.md]: it's not a breaking change

File: modules/vba_documents/lib/vba_documents/upload_error.rb (hunk 3 above)
Comment (ATeal) [reply modules/vba_documents/lib/vba_documents/upload_error.rb]: fair enough I'll add it there.

**Pull request merged**